src/components/User/Question.vue

```diff
@@ -65,25 +70,20 @@ export default {
             counter: 0,
             number:this.$route.query.number,
             correctnumber:this.$route.query.correctnumber,
-            nooption:false
         }
     },
     methods: {
         toScore: function() {
             this.$router.push({name:'score',query:{id:this.id,score:this.score}})
         },
         next: function() {
-            if (this.nooption==true) {
-                 this.nooption=false
-                 return
-            }
```

Comment:
I think you cannot remove these lines, otherwise the question number will keep increasing even when the user didnt choose any option